files/root/grafana_tools/ldap_sync/config

```diff
@@ -0,0 +1,3 @@
+base_dir=""
```

Comment:
Suggest: add a comment that this file is managed by Puppet

Reply:
Again, these files come from another git repo. I don't want to start editing or recreating the files.
But a great idea for a future improvement.